body.tex

```diff
@@ -2,7 +2,7 @@ \section{Introduction} \label{sec:intro}
 
 There is a perceived  risk that LSST  images may be snooped during transmission from the telescope. This concerns the images used for alert processing which are transmitted within a minute and provide a valuable resource for identifying moving objects including satellites. Though our processing will ignore satellites other parties may be interested in this information.
 
-There have been several communications concerning encryption on the wire this done into in some detail in \citeds{DMTN-163}.
+There have been several communications concerning encryption on the wire this gone into in some detail in \citeds{DMTN-163}.
```

Comment:
Suggest "There have been several communications concerning encryption on the wire, and this is covered in some detail in \citeds{DMTN-163}."